pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -251,7 +251,7 @@ func NewCertSecrets(installNamespace string, operatorNamespace string) []*k8sv1.
 // nextRotationDeadline returns a value for the threshold at which the
 // current certificate should be rotated, 80% of the expiration of the
 // certificate.
-func NextRotationDeadline(cert *tls.Certificate, ca *tls.Certificate, duration *metav1.Duration) time.Time {
+func NextRotationDeadline(cert *tls.Certificate, ca *tls.Certificate, duration *metav1.Duration, renewBefore *metav1.Duration, caOverlapTime *metav1.Duration) time.Time {
```

Comment:
`duration` seems to be unused, is it obsolete, should it be removed?

Reply:
:facepalm: good catch. Yes, the most correct action is to remove the parameter. thanks!